extensions/wikidata/src/org/openrefine/wikidata/schema/WbItemDocumentExpr.java

```diff
@@ -23,20 +23,20 @@
  ******************************************************************************/
 package org.openrefine.wikidata.schema;
 
-import java.util.Collections;
-import java.util.List;
-
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
Any chance of convincing your IDE to not reorder imports? It makes for a very noisy diff.